pkg/util/test/util.go

```diff
@@ -643,6 +644,71 @@ func validateImageTag(tag, namespace string, listOptions map[string]string) erro
 	return nil
 }
 
+func validateMonitoring(cluster *corev1alpha1.StorageCluster, timeout, interval time.Duration) error {
+	if cluster.Spec.Monitoring != nil &&
+		(*cluster.Spec.Monitoring.EnableMetrics ||
+			(cluster.Spec.Monitoring.Prometheus != nil && cluster.Spec.Monitoring.Prometheus.ExportMetrics)) {
+		if cluster.Spec.Monitoring.Prometheus != nil && cluster.Spec.Monitoring.Prometheus.Enabled {
+			dep := appsv1.Deployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "prometheus-operator",
```

Comment:
Name should be - px-prometheus-operator
Can we use the name constants from here? That way you don't have to hard code it?
https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/component/prometheus.go#L41